modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -85,6 +85,37 @@
 - t3a.2xlarge (8 vCPU, 32 GiB)
 ====
 
+.Memory-intensive
+[%collapsible]
+====
+- x1.16xlarge (64 vCPU, 976GiB)
```

Comment:
In the other tables, the number and GiB has a space in between them. Like the one on line 116